lib/docs/scrapers/markdown.rb

```diff
@@ -2,7 +2,8 @@ module Docs
   class Markdown < UrlScraper
     self.name = 'Markdown'
     self.type = 'simple'
-    self.base_url = 'http://daringfireball.net/projects/markdown/syntax'
+    self.release = '1.0.1'
+    self.base_url = 'https://daringfireball.net/projects/markdown/syntax'
```

Comment:
I don't think the Markdown specs are versioned. AFAIK, version 1.0.1 refers to the Perl reference implementation…